docs/developing/app-startup.md

```diff
@@ -0,0 +1,75 @@
+# Ways in which xPub Starts
+
+## Dockerfile(s)
+
+Dockerfiles don't specify the way in which the application starts (see below).
+
+```
+~/dev/elife-xpub $ grep CMD Dockerfile*
+Dockerfile:CMD []
+Dockerfile-development:CMD []
+```
+
+However, because the `Dockerfile` is meant for developers it does run the application
```

Comment:
That
- doesn't run the application: it is a command executed at image build time
- it is not meant for developers, `Dockerfile-development` is?